components/chef-ui-library/src/molecules/chef-phat-radio/chef-phat-radio.tsx

```diff
@@ -47,6 +47,14 @@ export class ChefPhatRadio {
       this.value = this.selected.value;
     }
     this.selected.selected = true;
+
+    // add keypress listeners to all the child options
+    const theseOptions = this.el.querySelectorAll(`chef-option`);
+    theseOptions
+      .forEach(option => {
+        option.addEventListener('keypress', event => { this.handleKeypress(event); });
+        option.setAttribute('tabindex', '0');
+      });
```

Comment:
In order to _not_ target all options across automate, I opted to only select the options that are inside the selected element -> in this case `chef-phat-radio`.

A tab index adds support for keyboard tabbing, and keypress event which routes the same function as the click listener.

Reply:
Is `theseOptions` needed? Seems like `options` would contain the same list of `chef-option` nodes, albeit as an `Array` instead of `NodeList`.

Reply:
thanks!